README.md

```diff
@@ -1,42 +1,84 @@
-# Split Android SDK
+# Split SDK for Android
+[![Build Status](https://travis-ci.com/splitio/android-client.svg?branch=master)](https://travis-ci.com/splitio/android-client)
+ 
+## Overview
+This SDK is designed to work with Split, the platform for controlled rollouts, which serves features to your users via a Split feature flag to manage your complete customer experience.
+ 
+[![Twitter Follow](https://img.shields.io/twitter/follow/splitsoftware.svg?style=social&label=Follow&maxAge=1529000)](https://twitter.com/intent/follow?screen_name=splitsoftware)
+ 
+## Compatibility
+This SDK is compatible with Android SDK versions 15 and later (4.0.3 Ice Cream Sandwich).
+
+## Getting started
+Below is a simple example that describes the instantiation and most basic usage of our SDK:
+```
+<Factory instantiation>
+<Block until Ready>
+<Get treatment>
+<If treatment == ...>
+```
```

Comment:
Same as for iOS, add the simplest code possible to "validate" the SDK behaviour. Use the official docs as inspiration
https://help.split.io/hc/en-us/articles/360020343291-Android-SDK#2-instantiate-the-sdk-and-create-a-new-split-client
https://help.split.io/hc/en-us/articles/360020343291-Android-SDK#basic-usage